web/src/assets/locales/en.json

```diff
@@ -205,6 +205,66 @@
     }
   },
 
+  "org": {
+    "settings": {
+      "settings": "Settings",
+      "not_allowed": "Not allowed to access this organization's settings",
+
+      "secrets": {
+        "secrets": "Secrets",
+        "desc": "Organization secrets can be passed to all organization's repository individual pipeline steps at runtime as environmental variables.",
+        "none": "There are no organization secrets yet.",
+        "add": "Add secret",
+        "save": "Save secret",
+        "show": "Show secrets",
+        "name": "Name",
+        "value": "Value",
+        "deleted": "Organization secret deleted",
+        "created": "Organization secret created",
+        "saved": "Organization secret saved",
+
+        "images": {
+          "images": "Available for following images",
+          "desc": "Comma separated list of images where this secret is available, leave empty to allow all images"
+        },
+        "events": {
+          "events": "Available at following events",
+          "pr_warning": "Please be careful with this option as a bad actor can submit a malicious pull request that exposes your secrets."
+        }
+      }
+    }
+  },
+
+  "admin": {
```

Comment:
I am not sure if it is a good idea to duplicate all of those values, although I like how you simply pass `i18n-prefix` to get the correct translations.

Reply:
Not all are duplicating and I did not like to having repo prefix in views that are not repo specific. I could probably move some out to some specific section maybe?

Reply:
Linking them sounds great. I had a look at the docs and its actually possible: https://kazupon.github.io/vue-i18n/guide/messages.html#linked-locale-messages

Reply:
I tried to link the translations. Which worked in general, but as just a few were duplicated it was more work than benefit in the end somehow.